shared/chat/conversation/info-panel/index.tsx

```diff
@@ -1,36 +1,61 @@
 import * as React from 'react'
 import * as Types from '../../../constants/types/chat2'
-import {Box, Divider, HeaderOnMobile, List} from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Kb from '../../../common-adapters'
+import * as RPCChatTypes from '../../../constants/types/rpc-chat-gen'
 // @ts-ignore
 import {Props as HeaderHocProps} from '../../../common-adapters/header-hoc/types'
-import {globalColors, globalMargins, globalStyles, isMobile, platformStyles} from '../../../styles'
-import {SmallTeamHeader, BigTeamHeader} from './header'
-import Notifications from './notifications/container'
-import AddPeople from './add-people'
+import {AdhocHeader, TeamHeader} from './header'
+import {SettingsPanel} from './panels'
 import Participant from './participant'
-import {ParticipantCount} from './participant-count'
-import {CaptionedButton, CaptionedDangerIcon} from './channel-utils'
-import RetentionPicker from '../../../teams/team/settings-tab/retention/container'
-import MinWriterRole from './min-writer-role/container'
+import {AttachmentTypeSelector, DocView, LinkView, MediaView} from './attachments'
 
-const border = `1px solid ${globalColors.black_10}`
-const listStyle = {
-  ...globalStyles.flexBoxColumn,
-  alignItems: 'stretch',
-  flex: 1,
-  paddingBottom: globalMargins.medium,
-  ...(isMobile
-    ? {}
-    : {
-        backgroundColor: globalColors.white,
-        borderLeft: border,
-      }),
+export type Panel = 'settings' | 'members' | 'attachments'
+
+type Thumb = {
```

Comment:
Maybe there should be a common `Thumb` type.

Reply:
Outside of this file?